task-1/app.py

```diff
@@ -0,0 +1,97 @@
+from flask import Flask, jsonify, request
+import json
+import http
+
+class InterfaceAPI(Flask):
+
+    FILE_NAME = "data.json"
+    FILTERED_OPTIONS = ["name", "type", "enabled"]
+
+    def __init__(self, import_name):
+        super(InterfaceAPI, self).__init__(import_name)
+        self.cache = self.read_json(self.FILE_NAME) 
+
+    def read_json(self, file_name):
+        with open(file_name) as f:
+            data = json.loads(f.read())
```

Comment:
@tomowex don t use json.loads and read... json has also method load. try and read about it but basicaly it reads from the file.. so all you need to do is give it an input file..
```
data = json.load(f)
```